subsys/bluetooth/host/Kconfig

```diff
@@ -23,6 +23,13 @@ config BT_LONG_WQ_PRIO
 	int "Long workqueue priority. Should be pre-emptible."
 	default 10
 	range 0 NUM_PREEMPT_PRIORITIES
+
+config BT_LONG_WQ_INIT_PRIO
+	int "Long workqueue init priority"
+	default 50
```

Comment:
The previous default was 40, and now it is being changed to 50. Is that intentional? What is the rationale behind setting it to 50 instead of 40?

Reply:
Yes, it is so that flash drivers can start which allows for relocating bluetooth code to an off-chip memory.

Reply:
https://github.com/zephyrproject-rtos/zephyr/pull/62449/files#diff-2022ef2d0357576be08373853a99f8e4aabe2b7607f4420d3f9ea8ccd5f9a7c8R18

Reply:
Thanks - Could we put that as part of the commit message? It's nice to have reasons for changes in the commit message :)

Reply:
> Could we put that as part of the commit message? It's nice to have reasons for changes in the commit message :)

Agreed. One can almost always answer the question “what?” by looking at the diff, however “why?” is not always clear and the commit message is the best place for this info.